third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -335,7 +335,7 @@ const TEST_CONFIGS: Lazy<BTreeMap<&str, TestConfig>> = Lazy::new(|| {
             include: vec!["/acquires-checker/"],
             exclude: vec![],
             exp_suffix: None,
-            options: opts.clone(),
+            options: opts.clone().set_experiment(Experiment::ACCESS_CHECK, false),
```

Comment:
skip access check for acquires-checker so that the error messages remain the same.

Reply:
Could you add this as a comment please?